_includes/specials/taproot/en/18-trivia.md

```diff
@@ -0,0 +1,83 @@
+{% auto_anchor %}
+
+- **What is a taproot?** Wikipedia [says][wikipedia taproot], "A taproot
+  is a large, central, and dominant root from which other roots sprout
+  laterally. Typically a taproot is somewhat straight and very thick, is
+  tapering in shape, and grows directly downward.  In some plants, such
+  as the carrot, the taproot is a storage organ so well developed that
+  it has been cultivated as a vegetable."
+
+    Bitcoin's [taproot][topic taproot], similar to the plant root, hides
+    the details of its scripts below the surface, away from public view,
+    and is very efficient at storing those scripts.
+
+- **Schnorr predates ECDSA:** we talk about [schnorr signatures][topic
+  schnorr signatures] as an upgrade on Bitcoin's original ECDSA
+  signatures because they make it easier to implement various
+  cryptographic tricks, but the schnorr signature algorithm predates the
+  DSA algorithm that ECDSA is based upon.  Indeed, DSA was created in
+  part to circumvent the [patent on schnorr signatures][schnorr patent]
+  and it was [reported][gov report] that "there were allegations of
+  infringement from Mr. Claus Schnorr and Public Key Partners (PKP)"
+  that DSA violated Schnorr's patent.  That patent expired in 2008.
+
+- **Un-Schnorr what name to use:** although it didn't work out, there
```

Comment:
In  fact, I think we did consider calling BIP340 "DLS" for "Discrete Logarithm Signatures", but I think we eventually reneged on that because the name Schnorr was already so much talked about.

Reply:
In light of my comment above, "Un-Schnorr" is another term I would personally avoid.